scss/_variables.scss

```diff
@@ -773,72 +764,62 @@ $font-sizes: (
-$h6-spacing:                  $letter-spacing-base !default;
-// scss-docs-end letter-spacing
-
-// stylelint-disable function-disallowed-list
-// scss-docs-start line-height
-$h1-line-height:              1 !default;
-$h2-line-height:              calc(32 / 30) !default;
-$h3-line-height:              calc(26 / 24) !default;
-$h4-line-height:              calc(22 / 20) !default;
-$h5-line-height:              calc(20 / 18) !default;
-$h6-line-height:              $line-height-base !default;
-// scss-docs-end line-height
-// stylelint-enable function-disallowed-list
-
 // scss-docs-start headings-variables
-$headings-margin-bottom:      $spacer !default; // OUDS mod
+$headings-margin-bottom:      $ouds-space-fixed-medium !default; // OUDS mod: instead of `$spacer * .5`
 $headings-font-family:        null !default;
 $headings-font-style:         null !default;
-$headings-font-weight:        700 !default;
-$headings-line-height:        $h6-line-height !default;
+$headings-font-weight:        $ouds-font-weight-heading !default;
+$headings-line-height:        null !default;
 $headings-color:              inherit !default;
 // scss-docs-end headings-variables
 
 // scss-docs-start display-headings
-$display1-size:               $font-size-xlg * 3 !default;        // 60px
-$display2-size:               $font-size-xlg * 2.5 !default;      // 50px
-$display3-size:               $font-size-xlg * 2 !default;        // 40px
-$display4-size:               $h1-font-size !default;             // 34px
-$display1-spacing:            $letter-spacing-base * 20 !default; // -2px
-$display2-spacing:            $letter-spacing-base * 16 !default; // -1.6px
-$display3-spacing:            $h1-spacing !default;               // -1px
-$display4-spacing:            $h1-spacing !default;               // -1px
-$display-line-height:         $h1-line-height !default;
+$display-font-sizes: (
+  large: "display-large",
+  medium: "display-medium",
+  small: "display-small"
+) !default;
+
+@if $enable-bootstrap-compatibility {
+  $display-font-sizes: map-merge(
+    $display-font-sizes,
+    (
+      1: "display-large",
+      2: "display-medium",
+      3: "display-small",
+      4: "heading-xlarge",
+      5: "heading-large",
+      6: "heading-medium",
+    )
+  );
+}
+
+$display-font-family: null !default;
+$display-font-style:  null !default;
+$display-font-weight: $ouds-font-weight-display !default;
+$display-line-height: $headings-line-height !default;
 // scss-docs-end display-headings
 
 // scss-docs-start type-variables
-$lead-font-size:              $font-size-xlg !default;
-$lead-font-weight:            400 !default;
-$lead-line-height:            1.5 !default;
-$lead-letter-spacing:         $letter-spacing-base * 4 !default;
+$lead-font-size:              $h5-font-size !default;
+$lead-font-weight:            $ouds-font-weight-body-default !default;
 
-$small-font-size:             .875rem !default;
+$small-font-size:             .875em !default;
```

Comment:
Are we sure this is a good idea, compared to the `.small` class that is in ` rem`?

Reply:
We removed the variable for now and replaced all its calls by `null`.